terraform/ecs/main.tf

```diff
@@ -328,6 +328,7 @@ module "validator" {
   account_id = data.aws_caller_identity.current.account_id
 
   ecs_context_json = jsonencode({
+    ecsClusterName : module.ecs_cluster.cluster_name
```

Comment:
One of the reason I added this is because the [expected metric template ](https://github.com/aws-observability/aws-otel-test-framework/blob/terraform/validator/src/main/resources/expected-data-template/ecsContainerExpectedMetric.mustache#L2-L7) needs to get the metrics from the container context. However, the current context does not have that. Therefore, it returns NullPointerException whenever variable sample_app_callable return true.